org.osgi.test.cases.servlet.secure/build.gradle

```diff
@@ -13,12 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * SPDX-License-Identifier: Apache-2.0 
- *******************************************************************************/
+ * SPDX-License-Identifier: Apache-2.0
+ */
 
-package org.osgi.test.cases.http.whiteboard.junit;
-
-public class SignatureTestCase extends
-		org.osgi.test.support.signature.SignatureTestCase {
-	// concrete subclass
-}
+/*
```

Comment:
I don't think we need this build.gradle file anymore since we now build and test on JDK 17.